msys2-ci-build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euo pipefail
+
+# Set default options
+BUILD_TYPE="Release"
+BUILD_DIR="build"
+INSTALL_DIR="$(pwd)/install"
+REGRESSION_TESTS="OFF"
+BUILD_SHARED_LIBS="ON"
+
+SYSTEMC_CI_DEFAULT_TARGET="systemc"
+SYSTEMC_CI_REGRESSION_TARGET="check-tests"
```

Comment:
The current docker entrypoint.sh does:
```
cmake
cmake --build
cmake --install
make ... check
```

Is this doing the same?
If so, I am in favor of unifying this accross workflows so that all CI flows use the same build targets.